index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Apple</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/mailtoharshit/San-Francisco-Font-/sanfrancisco.css">
+</head>
+<body>
+
+      
+    <header class="header">
+        <nav class="nav-bar">
+            <div class="nav-bar__item logo">
+              <a href="#">
+                <img src="https://upload.wikimedia.org/wikipedia/commons/1/1b/Apple_logo_grey.svg" alt="apple logo" class="apple-logo">
+              </a>
+            </div>
+        
+            <ul class="nav-bar__list">
+              <li class="nav-bar__list-item nav-bar__item"><a href="#" class="nav-bar__link">Store</a></li>
+              <li class="nav-bar__list-item nav-bar__item"><a href="#" class="nav-bar__link">Mac</a></li>
+              <li class="nav-bar__list-item nav-bar__item"><a href="#" class="nav-bar__link">iPod</a></li>
+              <li class="nav-bar__list-item nav-bar__item"><a href="#" class="nav-bar__link">iPhone</a></li>
+              <li class="nav-bar__list-item nav-bar__item"><a href="#" class="nav-bar__link">iPad</a></li>
+              <li class="nav-bar__list-item nav-bar__item"><a href="#" class="nav-bar__link">iTunes</a></li>
+              <li class="nav-bar__list-item nav-bar__item"><a href="#" class="nav-bar__link">Support</a></li>
+            </ul>
+        
+            <div class="nav-bar__item search-box">
+              <input type="search" class="search-input">
+            </div>
+        </nav>
+        
+        <section class="home hero">
+            <h1 class="title-hero">
+              What will your verse be?
+            </h1>
+            <div class="description">
+              <p class="copy">
+                Everyone has something to share. See how people are using iPad in their unique ways to add to the worlds's story.
+              </p>
+              <a href="#" class="call">Watch the "Your verse" film and explore this stories<span class="play-icon">
+                <img src="assets/play-button.svg" alt="play" class="play">
+              </span></a>
+              
+            </div>
+        </section>
+    </header>
+
+
+      
+      
+        <aside class="aside">
+          <a href="#">
+            <img src="assets/promo_iphone5s.jpg" alt="iPhone 5 S" class="aside-img">
+          </a>
+          <a href="#">
+            <img src="assets/promo_iphone5c.jpg" alt="iPhone 5 C" class="aside-img">
+          </a>
+          <a href="#">
+            <img src="assets/promo_30_years.jpg" alt="Happy Birthday, Mac" class="aside-img">
+          </a>
+          <a href="#">
+              <img src="assets/promo_supplier_responsibility.jpg" alt="Responsability" class="aside-img">
+          </a>
+        </aside>
+      
```

Comment:
Good job :+1: 

**[OPTIONAL]** Maybe a structure in which the images would be the background image of `<div>` containers would give you more freedom when adding changes down the road.